### ISSUE TITLE
web.config is different between `dotnet new webconfig` and `dotnet publish`

### ISSUE THREAD

**Issue by John0King** [aspnet/AspNetCore#4035]
dotnet new webconfig
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <!-- To customize the asp.net core module uncomment and edit the following section. 
  For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
  <!--
  <system.webServer>
    <handlers>
      <remove name="aspNetCore"/>
      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
    </handlers>
    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
  </system.webServer>
  -->

</configuration>

```

and in `dotnet publish`
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <location path="." inheritInChildApplications="false">
    <system.webServer>
      <handlers>
        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
      </handlers>
      <aspNetCore processPath="dotnet" arguments=".\webconfigtest.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
    </system.webServer>
  </location>
</configuration>
```

Is the web.config of `dotnet new webconfig` deprecated ?
Please unify the web.config template.

**Comment by jkotalik** [comment]
@John0King I noticed this too. Thanks for filing.

@vijayrkn I believe this issue is already filed in aspnet/websdk but wanted to confirm before closing.

**Comment by John0King** [comment]
@jkotalik   and the doc https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/aspnet-core-module?view=aspnetcore-2.1
I never seen/used  the `location` element in web.config, Is this element come with ANCM ?

**Comment by anurse** [comment]
I don't see something on aspnet/websdk already. @vijayrkn any thoughts? I can transfer this issue to the websdk repo if there isn't already something tracking it.

**Comment by vijayrkn** [comment]
@seancpeters - Do you know where is the code for "dotnet new web.config"? It should get updated to have the location tag since that is what publish and Run adds. 

/cc @BillHiebert

**Comment by vijayrkn** [comment]
@anurse - This issue should go to where ever we have the dotnet new template for web.config. I will wait for @seancpeters to confirm.

**Comment by anurse** [comment]
Looks like it's in [dotnet/templating](https://github.com/dotnet/templating/blob/stabilize/template_feed/Microsoft.DotNet.Common.ItemTemplates/content/WebConfig/web.config).

So the change we're aiming for is to add the `<location>` element to the commented out code in the `dotnet new` template?

**Comment by seancpeters** [comment]
@vijayrkn @anurse - If you want to make a PR for this, we can get it into the repo.

**Comment by vijayrkn** [comment]
@John0King - dotnet/templating is an open-source repo. We happily accept PRs from the community if you want to update the template.

**Comment by anurse** [comment]
https://github.com/dotnet/templating/pull/1887

**Comment by anurse** [comment]
Resolved in dotnet/templating#1887!

**Comment by anurse** [comment]
Accepted as this was external.
